CHANGELOG.md

```diff
@@ -1,126 +1,4 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
+[Visit](https://www.encode.io/databases/release-notes) to see the changelog.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
```

Comment:
I think the PR is getting way too big, for no reason. can we please stick only to necessary changes?
For example I think CHANGELOG.md is consistent withing all encode projects.